docs/source/quantization-support.rst

```diff
@@ -590,3 +590,7 @@ the `custom operator mechanism <https://pytorch.org/tutorials/advanced/torch_scr
 .. automodule:: torch.nn.quantizable.modules
 .. automodule:: torch.nn.quantized
    :noindex:
+.. automodule:: torch.ao.nn.quantized.reference
+   :noindex:
+.. automodule:: torch.ao.nn.quantized.reference.modules
```

Comment:
I feel we don't want to list this here, it's an implementation detail

Reply:
It asks me to either list here or makes it private. My understanding is they need to generate an implement doc for each public module and listing something here is to make a placeholder.

Reply:
I see, OK